SortingWithSOLID/src/main.cpp

```diff
@@ -1,23 +1,33 @@
-#include "../include/SortAnalyzer.h"
-#include "../include/Sorter.h"
-#include "../include/Logger.h"
-#include "../include/sorting.h"
+#include "./SortAnalyzer/SortAnalyzer.h"
+#include "./Sorter/Sorter.h"
+#include "./Logger/Logger.h"
+#include "./Sorting/sorting.h"
+
+const std::vector<int> randomGenerate(const int size)
+{
+	std::vector<int> vec;
+	for (int i = 0; i < size; ++i)
+	{
+		vec.push_back(std::rand() % 100);
```

Comment:
Please use modern random number generation algorithms which are defined in C++.

Why do we have restrictions on random number ranges?